cmd/dispenser/server.go

```diff
@@ -71,14 +71,24 @@ const topPageTemplate = `
     <script
       src="https://code.jquery.com/jquery-3.3.1.min.js"
       integrity="sha256-FgpCb/KJQlLNfOu91ta32o/NMZxltwRo8QtmkMRdAu8="
-      crossorigin="anonymous"></script>
-
-    <script>
+	   crossorigin="anonymous"></script>
+	  
+	 <script>
+	  function loadparm(){
+			const queryString = window.location.search;
+			const urlParams = new URLSearchParams(queryString);
+			const parmTarget = urlParams.get('account');
+			var target1 = document.getElementById('target');
+			target1.value = "";
+			target1.value = parmTarget;
+			var $target = $('target');
+			$target.html(target1.value);
```

Comment:
what's the purpose of having `target1` ? I think that you can do
$('#target').html(urlParams.get('account'))

Reply:
Yes, that is a cleaner way to do it @tsachiherman . Also, if urlParams.get('account') returns null it will be now setting the target.html to null which should be ok. Do I need to re-submit this? 

    const queryString = window.location.search;
    const urlParams = new URLSearchParams(queryString);
    $('#target').html(urlParams.get('account'))

Reply:
changed to $('#target').val  and this works.

Reply:
changed this
$('#target').html(urlParams.get('account'));
to this
$('#target').val(urlParams.get('account'));